airflow-core/tests/unit/api_fastapi/execution_api/conftest.py

```diff
@@ -31,9 +31,7 @@ def client(request: pytest.FixtureRequest):
     app = cached_app(apps="execution")
 
     # By specifying a "far-future" date, this will make the tests always run against the latest version
```

Comment:
@rawwar You left this comment around but removed the far future date it was talking about.

Fancy removing it too please?

Reply:
I will remove it.

Reply:
Done - https://github.com/apache/airflow/pull/48602